sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamEnumerableConverter.java

```diff
@@ -133,14 +219,108 @@ private static PipelineResult run(
         options
             .getRunner()
             .getCanonicalName()
-            .equals("org.apache.beam.runners.direct.DirectRunner"));
+            .equals("org.apache.beam.runners.direct.DirectRunner"),
+        "Only DirectRunner is supported in SQL Shell. " + "Please check your Runner setting.");
```

Comment:
This isn't exactly true. Should be something like "SELECT without INSERT is only supported in DirectRunner in SQL Shell."

Reply:
Fixed.